src/components/textarea/textarea.component.tsx

```diff
@@ -379,7 +400,9 @@ export const Textarea = React.forwardRef(
             >
               {characterLimit || inputHint ? (
                 <StyledInputHint id={hintIdValue} data-element="input-hint">
-                  {characterCountHint || inputHint}
+                  {formattedCharacterCountHint ||
```

Comment:
Question: do we need to have 3 string props for this? Couldn't this be achieved via a single `inputHint` string prop, if it needs to be formatted you could check if the `characterLimit` is set etc

```
const formattedHint = characterLimit ? inputHint?.replace(
      /\{.*?\}/g,
      formattedCharacterLimit
    ) : inputHint;
```

Reply:
Yeah, this is a good point, even though the intentions were good by making sure a dedicated hint was provided via the `characterCountHint` which supersedes an `inputHint`, I feel like we can just use the `inputHint` prop and trust consumers to do the right thing here